src/openforms/logging/templates/logging/admin_list.html

```diff
@@ -10,4 +15,19 @@
         </li>
     {% endfor %}
 </ul>
+
+{% if is_paginated %}
+    <div id="changelist">
+        <p class="paginator">
+            {% for i in paginator.page_range %}
+                {% if page_obj.number == i %}
+                    <span class="this-page">{{ i }}</span>
+                {% else %}
+                    <a href="?page={{ i }}" class="end">{{ i }}</a>
+                {% endif %}
+            {% endfor %}
+            {{ paginator.count }} {% if paginator.count == 1 %}{% trans "timeline log entry" %}{% else %}{% trans "timeline log entries" %}{% endif %}
```

Comment:
I couldn't find a way to access the model's verbose name's in the template, since it's stored in `_meta`.  All the suggestions I saw online mentioned adding a template tag.  Not sure if we want that or if this is okay?

Reply:
I can live with it :P the django admin typically puts `Model._meta` in the context as `opts`.

However, the correct django approach is to use the `{% plural %}` tag in `{% blocktrans %}`, see https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#blocktranslate-template-tag